server/src/routes/watchlist.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const WatchLists = require('../db/mongo/models/watchlist');
+
+const router = express.Router();
+
+router.get('/:username', async (req, res) => {
+    const username  = req.params.username;
+    try {
+        let watchlist = await WatchLists.find({ username: username}).exec();
```

Comment:
As we spoke , use my method's `getUserWatchlist` when it's approved.
Also, `/:email` and not `/:username` (`watchlist` model is edited as well)